docs/user_manual/source/rvfi.rst

```diff
@@ -230,13 +230,18 @@ For |corev|, the ``rvfi_mem`` interface has been expanded to support multiple me
    output [NRET * NMEM * XLEN - 1 : 0]   rvfi_mem_rdata
    output [NRET * NMEM * XLEN - 1 : 0]   rvfi_mem_wdata
    output [NRET * NMEM * 3    - 1 : 0]   rvfi_mem_prot
+   output [NRET * NMEM * 6    - 1 : 0]   rvfi_mem_atop
+   output [NRET * NMEM * 1    - 1 : 0]   rvfi_mem_err
+   output [NRET * NMEM * 1    - 1 : 0]   rvfi_mem_exokay
+   output [NRET * NMEM * 2    - 1 : 0]   rvfi_mem_memtype
 
 Instructions will populate the ``rvfi_mem`` outputs with incrementing ``NMEM``, starting at ``NMEM=1``.
 
 Instructions with a single memory operation (e.g. all RV32I instructions), including split misaligned transfers, will only use NMEM = 1.
 Instructions with multiple memory operations (e.g. the push and pop instructions from Zcmp) use NMEM > 1 in case multiple memory operations actually occur.
 ``rvfi_mem_prot`` indicates the value of OBI prot used for the memory access or accesses. Note that this will be undefined upon access faults.
-
+``rvfi_mem_memtype`` indicates the memory type attributes associated with each memory operation (i.e cacheable or bufferable).
+``rvfi_mem_atop`` indicates the type of atomic transaction as specified in [OPENHW-OBI]_.
```

Comment:
And how about rvfi_mem_err and rvfi_mem_okay? What are they for and for what transactions do they actually carry valid information? See also https://github.com/openhwgroup/cv32e40x/pull/947 and my comments on that one.

Reply:
Added explanation for err and exokay as suggested in #947.